<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SectorWeekAnalysis</class>
 <widget class="QWidget" name="SectorWeekAnalysis">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>201</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QComboBox" name="comboBoxSelectSector">
     <property name="minimumSize">
      <size>
       <width>181</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>Seleccione Categoría:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QGroupBox" name="groupBoxVisType">
     <property name="minimumSize">
      <size>
       <width>140</width>
       <height>90</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>160</width>
       <height>90</height>
      </size>
     </property>
     <property name="title">
      <string>Tipo de visualización</string>
     </property>
     <widget class="QRadioButton" name="radioButtonAverageDisag">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Promedio desagregado</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButtonAverage">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Promedio total</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QComboBox" name="comboBoxSelectSubCategory">
     <property name="minimumSize">
      <size>
       <width>181</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>181</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
